docs/organizations/overview.mdx

```diff
@@ -5,26 +5,140 @@ description: Learn about how to create and manage Clerk organizations and their
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**. Here, you can view and manage all organizations within your application.
+3. Select a specific organization to view its details, members, and settings. Here, you can update the organization's name, slug, and logo. You can also set the organization's [membership limit](#how-many-organizations-can-i-create) and public and private metadata.
+
+For managing organizations in your application, Clerk provides a set of prebuilt components:
+
+- [`<CreateOrganization />`](/docs/components/organization/create-organization) - A form for a user to create a new organization.
+- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization. Out of the box, this component's **General** tab displays the organization's information and the **Leave organization** button. Admins will be able to see the **Update profile** button, **Verified domains** section, and **Delete organization** button. The **Members** tab shows the organization's members along with their join dates and roles. Admins will have the ability to invite a member, change a member's role, or remove them from the organization. Admins will have tabs within the **Members** tab to view the organization's invitations and requests.
+- [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) - A dropdown menu that allows a user to switch between their personal account and their organization account(s). Out of the box, this component will show notifications to the user if they have organization invitations or suggestions. Admins will be able to see notifications for requests to join an organization.
+- [`<OrganizationList />`](/docs/components/organization/organization-list) - A list of organizations that a user is a member of.
+
+If Clerk's prebuilt components don't meet your specific needs or if you require more control over the logic, you can rebuild the existing Clerk flows using the Clerk API. Currently, there are guides available for:
+
+- [Creating an organization](/docs/organizations/creating-organizations)
+- [Updating an organization](/docs/organizations/updating-organizations)
+- [Creating an infinite paginated list of memberships](/docs/organizations/viewing-memberships)
+- [Inviting users to an organization](/docs/organizations/inviting-users), which also includes code for creating a custom list of invitations
+- [Managing memberships](/docs/organizations/managing-roles), which includes code for updating and deleting a user's membership, for inviting a user, and for creating a custom list of memberships, invitations, and requests
+
+## Frequently asked questions (FAQ)
```

Comment:
This is not a blocking comment at all, but I'm personally anti FAQs in the docs in most (not all) scenarios. This [quote](https://medium.com/bts-design-team/heres-a-frequently-asked-question-for-you-are-faqs-all-bad-ebc37a746f12) summarizes it for me:

> [FAQs] duplicate information that — if it really matters to your users — should form part of a wider well-structured, user-centric journey on your site. They also create content overhead: an FAQ needs to be monitored and maintained and risks confusing users if you don’t keep it up to date. The problem of FAQs really shows in search, where you will end up with duplicate results competing for attention. You are fighting with your own content.

I'm aware we probably use FAQs across the docs, and our non-docs team members love them because they reduce the burden of having to think about content design and structure, but in future I'd like us to try to avoid them by asking "Where are users going to look for this information? How can I make sure it's there, instead of in one crowded place?"

Reply:
Good point ! We can remove it